src/app/components/core/developer/additional-services/additional-services-utils.service.ts

```diff
@@ -168,10 +162,16 @@ export class AdditionalServicesUtils {
           atIndex = idx;
         }
       } else {
+        let scheduleStatus;
+        if (foundSchedule) {
+          scheduleStatus = foundSchedule?.enabled === true ? 'running' : 'disabled';
+        }
+        let isFailedOrUnresponsiveService = ['failed', 'unresponsive'].includes(foundService.status);
```

Comment:
For south /north instances shutdown service state, do we show disabled or shutdown i.e. When /service return shutdown state i.e. Service's schedule was disabled and Fledge is not restarted?